specs/validator/tableland-openapi-spec.yaml

```
@@ -0,0 +1,304 @@
+      description: Returns version information about the validator.
+      operationId: version
+      responses:
+        "200":
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/VersionInfo"
+  /query:
+    get:
+      tags:
+        - query
+      summary: Query the Tableland network
+      description: Returns the results of a SQL read query against the Tabeland network
+      operationId: queryFromQuery
+      parameters:
+        - name: statement
+          in: query
+          description: The SQL read query statement
+          required: true
+          schema:
+            type: string
+        - name: format
+          in: query
+          description: >
+            The requested response format:
+             * `objects` - Returns the query results as a JSON array of JSON objects.
+             * `table` - Return the query results as a JSON object with columns and rows properties.
+          required: false
+          schema:
+            type: string
+            enum:
+              - objects
+              - table
+            default: "objects"
+        - name: extract
+          in: query
+          description: Whether or not to extract the JSON object from the single property of the surrounding JSON object.
+          required: false
+          schema:
+            type: boolean
+            default: false
+        - name: unwrap
+          in: query
+          description: Whether or not to unwrap the returned JSON objects from their surrounding array.
+          required: false
+          schema:
+            type: boolean
+            default: false
+      responses:
+        "200":
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                type: object
+        "400":
+          description: Invalid query/statement value
+  /receipt/{chainId}/{transactionHash}:
+    get:
+      tags:
+        - receipt
+      summary: Query the Tableland network for transaction status
+      description: Returns the status of a given transaction receipt by hash
+      operationId: receiptByTransactionHash
+      parameters:
+        - name: chainId
+          in: path
+          description: The parent chain to target
+          required: true
+          schema:
+            type: integer
+            format: int32
+        - name: transactionHash
+          in: path
+          description: The transaction hash to request
+          required: true
+          schema:
+            type: string
+      responses:
+        "200":
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/TransactionReceipt"
+        "400":
+          description: Invalid chain identifier or transaction hash format.
```

This one has a period, but others don't... 
```suggestion
          description: Invalid chain identifier or transaction hash format
```